app/assets/stylesheets/application.css.scss

```diff
@@ -36,8 +36,8 @@
   position: relative;
 }
 
-.trophy-case-container {
-  background-image: url('http://urlm.co/blog/wp-content/uploads/ipad23.jpg');
+.yoga-container {
+  background-image: url('http://media.yogajournal.com/wp-content/uploads/245_hp_163.jpg');
```

Comment:
Prefer double-quoted strings<br>URL `http://media.yogajournal.com/wp-content/uploads/245_hp_163.jpg` should not contain protocol or domain